WISE-Community/WISE-API
# build(Spring-Boot): Upgrade from 2.3.2 to 2.4.13

## Changes
- Upgrade jedis 3.1.0->3.7.0 and restarting Redis
- Break up websocket endpoints into their own controllers

## Test
- Run mvn clean compile install to download dependencies
- Restart redis to remove any existing sessions
- Make sure that basic functionalities work
- Make sure that websocket features work: pause/unpause screens, lock/unlock lessons

Closes #70
